e2e/libs/utility/utility.py

```diff
@@ -107,6 +129,246 @@ def wait_delete_ns(name):
         time.sleep(RETRY_INTERVAL)
     assert not found
 
+def create_deployment(filepath):
```

Comment:
if needed, can create other modules (in this case, it could be workload.py) to organize util functions to prevent a long implementation in util.py.

Reply:
moved to `workload.py`.